src/vm/ClrEtwAll.man

```diff
@@ -6803,7 +6846,16 @@
                 <string id="RuntimePublisher.TieredCompilationResumeEventMessage" value="ClrInstanceID=%1;%nNewMethodCount=%2" />
                 <string id="RuntimePublisher.TieredCompilationBackgroundJitStartEventMessage" value="ClrInstanceID=%1;%nPendingMethodCount=%2" />
                 <string id="RuntimePublisher.TieredCompilationBackgroundJitStopEventMessage" value="ClrInstanceID=%1;%nPendingMethodCount=%2;%nJittedMethodCount=%3" />
-
+                <string id="RuntimePublisher.ResolutionAttempted.FindInLoadContext" value="NONE" />
+                <string id="RuntimePublisher.ResolutionAttempted.AssemblyLoadContextLoad" value="NONE" />
+                <string id="RuntimePublisher.ResolutionAttempted.PlatformAssemblies" value="NONE" />
+                <string id="RuntimePublisher.ResolutionAttempted.DefaultAssemblyLoadContextFallback" value="NONE" />
+                <string id="RuntimePublisher.ResolutionAttempted.AssemblyLoadContextResolvingEvent" value="NONE" />
+                <string id="RuntimePublisher.ResolutionAttempted.AppDomainAssemblyResolveEvent" value="NONE" />
```

Comment:
I think `RuntimePublisher.ResolutionAttemptedStage.*` and `RuntimePublisher.ResolutionAttemptedResult.*` would be more consistent with other strings for value maps.